docs/products/kafka/kafka-mirrormaker/howto/exactly-once-delivery.md

```diff
@@ -0,0 +1,191 @@
+  exactly-once delivery.
+
+</TabItem>
+
+<TabItem value="api" label="Aiven API">
+
+Use the [ServiceKafkaMirrorMakerCreateReplicationFlow](https://api.aiven.io/doc/#tag/Service:_Kafka_MirrorMaker/operation/ServiceKafkaMirrorMakerCreateReplicationFlow) API to create or update a replication flow
+with `exactly_once_delivery_enabled` in the configuration.
+
+- Create a new replication flow with exactly-once delivery enabled:
+
+  ```bash
+  curl -X POST "https://console.aiven.io/v1/project/<project_name>/service/<service_name>/mirrormaker/replication-flows" \
+     -H "Authorization: Bearer <API_TOKEN>" \
+     -H "Content-Type: application/json" \
+     -d '{
+           "source_cluster": "<source_cluster>",
+           "target_cluster": "<target_cluster>",
+           "enabled": true,
+           "exactly_once_delivery_enabled": true
+         }'
+  ```
+
+- Update an existing replication flow to enable exactly-once delivery:
+
+  ```bash
+  curl -X PUT "https://console.aiven.io/v1/project/<project_name>/service/<service_name>/mirrormaker/replication-flows/<source_cluster>/<target_cluster>" \
+     -H "Authorization: Bearer <API_TOKEN>" \
+     -H "Content-Type: application/json" \
+     -d '{
+           "exactly_once_delivery_enabled": true
+         }'
+  ```
+
+- Update an existing replication flow to disable exactly-once delivery:
+
+  ```bash
+  curl -X PUT "https://console.aiven.io/v1/project/<project_name>/service/<service_name>/mirrormaker/replication-flows/<source_cluster>/<target_cluster>" \
+     -H "Authorization: Bearer <API_TOKEN>" \
+     -H "Content-Type: application/json" \
+     -d '{
+           "exactly_once_delivery_enabled": false
+         }'
+  ```
+
+Parameters:
+
+- `<project_name>`: Name of your project.
+- `<service_name>`: Name of your Aiven for Apache Kafka MirrorMaker 2 service.
+- `API_TOKEN`: Your personal Aiven API [token](/docs/platform/howto/create_authentication_token).
+- `<source_cluster>`: Alias of the source Aiven for Apache Kafka cluster.
+- `<target_cluster>`: Alias of the target Aiven for Apache Kafka cluster.
+- `exactly_once_delivery_enabled`:  Set to `true` to enable or `false` to disable
+  exactly-once delivery.
+
+</TabItem>
+<TabItem value="terraform" label="Terraform">
+
+Find information about the `aiven_mirrormaker_replication_flow` resource in the
+[Aiven Provider for Terraform® documentation](https://registry.terraform.io/providers/aiven/aiven/latest/docs/resources/mirrormaker_replication_flow).
+
+To enable exactly-once delivery:
+
+1. Update or create the `aiven_mirrormaker_replication_flow` resource in your
+   Terraform configuration file:
+
+   - Add `exactly_once_delivery_enabled = true` to the resource block.
+   - Specify the `topics` to replicate using patterns, such as `.*` for all topics.
+
+   ```hcl
+   resource "aiven_mirrormaker_replication_flow" "example_flow" {
+     project                       = var.project_name
```

Comment:
It might be better to add either an example value like "my-project" or use a placeholder (which would also match the other instructions) like:
```suggestion
     project                       = PROJEC_NAME
```

I think the placeholder would make it clearer what you need to replace with the project name, service name, etc., since these are right within the block as opposed to the variables, which we don't see defined here (understandably as they'd live in a different file usually).